pyproject.toml

```diff
@@ -146,13 +146,14 @@ exclude = ["tests/artifacts/**/*.safetensors", "*_pb2.py", "*_pb2_grpc.py"]
 # N: pep8-naming
 # TODO: Uncomment rules when ready to use
 select = [
-    "E", "W", "F", "I", "B", "C4", "T20", "N" # "SIM", "A", "S", "D", "RUF", "UP"
+    "ASYNC", "B", "C4", "E", "F", "I", "N", "PERF", "T20", "W" # "SIM", "A", "S", "D", "RUF", "UP"
```

Comment:
Keep these sorted to make it easy to spot missing rules and nearly impossible to add duplicates.